openwisp_radius/api/urls.py

```diff
@@ -62,6 +61,7 @@ def get_api_urls(api_views=None):
             api_views.change_phone_number,
             name='phone_number_change',
         ),
+        path('<slug:slug>/radiusbatch/', api_views.batch, name='batch'),
```

Comment:
Note the url is `/radiusbatch/` instead of `/radius/batch/` suggested in the issue.
This is because the other radiusbatch download url also has `/radiusbatch/` and I wanted to maintain the pattern.
If you think we should change it either in here or both places, please let me know! :smile:

Reply:
the change is fine to me, there are 2 issues:

- although adding `slug` here doesn't make much sense, because the other endpoints are designed to be used from the captive pages (or openwisp-wifi-login-pages). This endpoint is more like an API which could replace the django admin, so it's a bit different than the other ones.
- docs will need to be updated

Reply:
I think there is some confusion here:
1. `<slug:slug>/radiusbatch/` change is [mentioned in the issue](https://github.com/openwisp/openwisp-radius/issues/94), and when we [make a isOrganizationMember check in openwisp-users](https://github.com/openwisp/openwisp-users/issues/150) this would be required by that as well otherwise we will have to implement organization check for this endpoint separately. (my understanding is that `isOrganizationMember` is required regardless where the API is used when it's exposed to `staff/operator` users.) 
2. I have [already updated the API docs](https://github.com/openwisp/openwisp-radius/pull/131/files#diff-29397e8ff3afc883a7fb7f52c5a227e1R481), I don't see `/api/v1/batch/` anywhere else, am I missing any other file?

Reply:
you're right! I have wrote that but realized it may not be the best solution.
However, I don't think it's wise to invest a lot more time in this now. So it's good enough, thank you!